packages/server-wallet/src/handlers/join-channel.ts

```diff
@@ -1,14 +1,7 @@
-import {Either, left, right, chain, map} from 'fp-ts/lib/Either';
-import {StateVariables} from '@statechannels/wallet-core';
-import {pipe} from 'fp-ts/lib/function';
 import {ChannelId} from '@statechannels/client-api-schema';
 
-import {SignState} from '../protocols/actions';
-import {ChannelState} from '../protocols/state';
 import {WalletError, Values} from '../errors/wallet-error';
 
-type HandlerResult = Either<JoinChannelError, SignState>;
-type StepResult = Either<JoinChannelError, ChannelState>;
 export interface JoinChannelHandlerParams {
```

Comment:
New issue: This file ought to be deleted.

- [ ] If `JoinChannelHandlerParams` is used, it should be re-named and re-exported elsewhere.
- [ ] `JoinChannelError` should be re-exported elsewhere

Reply:
https://github.com/statechannels/statechannels/issues/2803